pydantic/fields.py

```diff
@@ -60,7 +62,6 @@ class FieldInfo(_repr.Representation):
         'allow_inf_nan': None,
         'min_items': None,
         'max_items': None,
-        'frozen': None,
```

Comment:
I'm not sure why `frozen` was in the metadata lookup like this, or if it needed to be; moving it to being an attribute of `FieldInfo` didn't make any tests fail though and made the necessary changes easier.